src/components/Capture/capture.js

```diff
@@ -80,8 +80,13 @@ class Capture extends Component {
   }
 
   handleMessages = (message) => {
-    const { actions } = this.props
-    const valid = message.valid;
+    const { actions, onError } = this.props
+    if (message.error) {
+      this.onServerError()
+      events.emit('onError')
```

Comment:
You could call `Tracker.sendError` directly here rather than using an event here.